api/proto/info/info.proto

```diff
@@ -631,6 +634,26 @@ enum MaintenanceModeReason {
   MAINTENANCE_MODE_REASON_LOW_DISK_SPACE = 3;
 }
 
+// Information about attestation process
+message AttestationInfo {
+  AttestationState state = 1;   //current state of attestation
+  ErrorInfo error = 2;          //last error
+}
+
+// Different states of attestation process
+// Must match the values in pkg/pillar/attest.State
+enum AttestationState {
+  ATTESTATION_STATE_NONE = 0;                    //State when (Re)Starting attestation
+  ATTESTATION_STATE_NONCE_WAIT = 1;              //Waiting for response from Controller for Nonce request
+  ATTESTATION_STATE_INTERNAL_QUOTE_WAIT = 2;     //Waiting for internal PCR quote to be published
+  ATTESTATION_STATE_INTERNAL_ESCROW_WAIT = 3;    //Waiting for internal Escrow data to be published
```

Comment:
It might be more clear to change INTERNAL to TPM. In the case of the quote we are definitely waiting for the TPM via the tpmmgr. In the encrypted key escrow that is probably also the case. Thus the comments can be written to reflect these details as well.

Reply:
Make sense, modified